### PR TITLE
Theme updates

### PR DESCRIPTION
**Pull request by jeffchang5**
Layout + some fragments
screen shot 2017-05-12 at 11 56 28 pm

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (connorkrupp) [inline .idea/vcs.xml]: This should probs be in gitignore

File: app/src/main/java/android/mhacks/org/mhacks_android/MainActivity.java [hunk 2]
```diff
@@ -16,13 +20,13 @@
         public boolean onNavigationItemSelected(@NonNull MenuItem item) {
             switch (item.getItemId()) {
                 case R.id.navigation_home:
-                    return true;
+                    break;
```
Comment (connorkrupp) [inline app/src/main/java/android/mhacks/org/mhacks_android/MainActivity.java]: Is this switch even doing anything

**Comment by connorkrupp** [comment]
I don't really know whats going on with Android nor did I try running this to see how it really looked, @shashankmurching should also look at this

File: .idea/vcs.xml (hunk 1 above)
Comment (jeffchang5) [inline .idea/vcs.xml]: These files are the configuration for the IDE itself. It just lets Android Studio know that Git is the VCS and I can commit from the IDE

**Pull request closed without merging**